multi-tenant/README.md

```diff
@@ -29,6 +29,6 @@ $ docker compose up
 If you have compiled a development distribution of Opencast on your host machine (using `-Pdev`), you can run this compiled version with the `docker-compose.dev.yaml` variation. Use the following commands to point to the Opencast source folder and start up the containers:
 
 ```sh
-$ expose OPENCAST_SRC="/path/to/my/opencast/src/root/folder"
-$ docker compose -f docker-compose.yaml -f docker-compose.dev.yaml up
+$ export OPENCAST_SRC="/path/to/my/opencast/src/root/folder"
+$ docker-compose -f docker-compose.yaml -f docker-compose.dev.yaml up
```

Comment:
This is not a typo. Compose has been integrated into the `docker` CLI: https://docs.docker.com/compose/reference/

```suggestion
$ docker compose -f docker-compose.yaml -f docker-compose.dev.yaml up
```

Reply:
Dang, I thought of exactly that and tried it locally and it didn't work. Should have googled instead of trusting my super old Ubuntu installation :grimacing: